azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -27,31 +27,31 @@ class GenericIoTHubClient(AbstractIoTHubClient):
     This class needs to be extended for specific clients.
     """
 
-    def __init__(self, transport):
+    def __init__(self, pipeline_adapter):
```

Comment:
As in the `abstract_clients.py` I question the semantics of this being called `pipeline_adapter`